developerportal/apps/events/migrations/0030_add_Extra_Content_panel_to_Event.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 2.2.14 on 2020-07-06 12:26
```

Comment:
This files' changes are from another PR which was based off this one, which I should have held off merging, but didn't... Sorry.

Ignore for review